src/wtforms/locale/es/LC_MESSAGES/wtforms.po

```diff
@@ -109,63 +109,63 @@ msgstr "Valor inválido, no puede ser ninguno de: %(values)s."
 
 #: src/wtforms/csrf/core.py:96
 msgid "Invalid CSRF Token."
-msgstr "El token CSRF es incorrecto"
+msgstr "El token CSRF es incorrecto."
 
 #: src/wtforms/csrf/session.py:63
 msgid "CSRF token missing."
-msgstr "El token CSRF falta"
+msgstr "El token CSRF falta."
 
 #: src/wtforms/csrf/session.py:71
 msgid "CSRF failed."
-msgstr "Fallo CSRF"
+msgstr "Fallo CSRF."
 
 #: src/wtforms/csrf/session.py:76
 msgid "CSRF token expired."
-msgstr "El token CSRF ha expirado"
+msgstr "El token CSRF ha expirado."
 
 #: src/wtforms/fields/core.py:534
 msgid "Invalid Choice: could not coerce."
-msgstr "Elección inválida: no se puede ajustar"
+msgstr "Elección inválida: no se puede ajustar."
 
 #: src/wtforms/fields/core.py:538
 msgid "Choices cannot be None."
-msgstr ""
+msgstr "La elección no puede ser None."
 
 #: src/wtforms/fields/core.py:545
 msgid "Not a valid choice."
-msgstr "Opción inválida"
+msgstr "Opción inválida."
 
 #: src/wtforms/fields/core.py:573
 msgid "Invalid choice(s): one or more data inputs could not be coerced."
 msgstr ""
-"Opción(es) inválida(s): una o más entradas de datos no pueden ser "
+"Opción(es) inválida(s): una o más entradas de datos no pueden ser coaccionadas."
 "ajustadas"
```

Comment:
It looks like you forgot to remove `ajustadas` :)

Reply:
Thanks for pointing it out. It is fixed!
Cheers!